flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/ExecutionGraphTestUtils.java

```diff
@@ -431,6 +433,18 @@ public static ExecutionJobVertex getExecutionVertex(JobVertexID id) throws Excep
 		return getExecutionVertex(id, new DirectScheduledExecutorService());
 	}
 
+	public static ExecutionEdge createMockExecutionEdge(int maxParallelism) {
```

Comment:
We are trying to get rid of Mockito in flink tests.
If we are touching this part, I'd prefer to introduce a mock ExecutionEdge class for this purpose.

Reply:
This is reused from ExecutionVertexDeploymentTest.

Let me have a try to see whether we can avoid Mockito and at the same time not to make the code too complicated.

Reply:
Removed Mockito.